### PR TITLE
:mortar_board: Topic n queens GA --- initialization section 

### PR DESCRIPTION
**Pull request by jameshughes89**
### What

Discuss the rep and creation of the chromosomes and population 

### Testing

:+1:

### CODE REVIEW

File: site/topics/genetic-algorithms/n-queens-ga.rst [hunk 1]
```diff
@@ -14,12 +14,61 @@ Problem
...
+
+.. literalinclude:: /../src/ga_n_queens.py
+    :language: python
+    :lineno-match:
+    :start-after: # [begin-initialization]
+    :end-before: # [end-initialization]
+
+
+* The number of chromosomes within the population is defined by a hyperparameter called ``POPULATION_SIZE``
+* The length of the chromosomes will match the number of queens, which is defined by the hyperparameter ``N_QUEENS``
```
Comment (twentylemon) [inline site/topics/genetic-algorithms/n-queens-ga.rst:65]: Isn't a hyperparameter multidimensional? Cardinality and N are only scalar.

**Review by twentylemon — APPROVED** [review]

File: site/topics/genetic-algorithms/n-queens-ga.rst (hunk 1 above)
Comment (jameshughes89) [reply site/topics/genetic-algorithms/n-queens-ga.rst:65]: The deep learning people started making "hyperparameter" mean a tunable parameter before the search starts as they use the word parameter for things like network weights that update during learning.

Comment (twentylemon) [reply site/topics/genetic-algorithms/n-queens-ga.rst:65]: ...k. Not sure why they'd ever choose a word that already means higher dimensional, when things like _variable_ and _argument_ already exist.

Comment (jameshughes89) [reply site/topics/genetic-algorithms/n-queens-ga.rst:65]: That's basically all of ML. They make up new words for things that already have a word. 

> At the beginning of this famous book "All of Statistics", the author has a table with the words the math/stats people use and the word the ML people use. It's amusing.

**Pull request merged**